src/types.ts

```diff
@@ -47,4 +47,6 @@ export type InstallationAccessTokenOptions = {
    * Find the app’s installation token at https://github.com/apps/<app name>/installations/new. Select the account then copy the number from the end of the URL
    */
   installationId: number;
+  repositoryIds?: number[];
+  permissions?: { [permission: string]: string };
```

Comment:
I think the only allowed values are `"read"` or `"write"`?

Reply:
good point. it depends on the permission but I added stricter types. i wasn't sure about the consistency between camel case (for `repositoryIds`) and snake case for the permissions

Reply:
camelCase is more JavaScript-y, so I default to that. GitHub’s APIs use camelcase though, so if things are passed through 1:1 then I use that. So in that case, I’d stick with GitHub’s naming of the permissions.

This is great, thank you! I’m a bit nervous about keeping it up-to-date now, but that’s something we can think about after this PR :)